src/app/body/milestone-details/milestone-details.component.ts

```diff
@@ -85,8 +89,10 @@ export class MilestoneDetailsComponent implements OnInit {
       this.getMilestoneDetails();
       this.milestoneService.getTasks(this.orgDomain, this.milestoneId);
       this.milestoneService.taskDataStateObservable.subscribe(()=>{
+        this.tasks = this.milestoneService.taskData;
         this.getNumberData();
         this.taskDataReady = true;
+        console.log("check3",this.milestoneService.taskData); //To Test in Production
```

Comment:
remove console

Reply:
Sanjay bhaiya said, its need to test in production.